src/serves/qiniu_cdn/datasource/comb/controllers.rs

```diff
@@ -0,0 +1,29 @@
+use axum::{extract::Path, Json};
+use ceobe_cookie_logic::{
+    impletements::CeobeCookieLogic, view::CombIdToCookieIdReq,
+};
+use http::StatusCode;
+use persistence::redis::RedisConnect;
+use resp_result::MapReject;
+use tracing::instrument;
+
+use super::error::QiniuCdnDatasourceCombError;
+use crate::router::QiniuCdnDatasourceCombFrontend;
+
+impl QiniuCdnDatasourceCombFrontend {
+    #[instrument(ret, skip(redis_client))]
+    // #[debug_handler]
+    pub async fn get_newest_cookie(
+        redis_client: RedisConnect,
+        MapReject(comb_id): MapReject<
+            Path<String>,
+            QiniuCdnDatasourceCombError,
+        >,
+    ) -> (StatusCode, Json<Option<CombIdToCookieIdReq>>) {
+        match CeobeCookieLogic::newest_comb_info(redis_client, comb_id).await
+        {
+            Ok(combid_res) => (StatusCode::OK, Some(combid_res).into()),
+            Err(_) => (StatusCode::from_u16(500).unwrap(), None.into()),
```

Comment:
```suggestion
            Err(_) => (StatusCode::INTERNAL_SERVER_ERROR, None.into()),
```

[INTERNAL_SERVER_ERROR](https://docs.rs/http/latest/http/status/struct.StatusCode.html#associatedconstant.INTERNAL_SERVER_ERROR)